embed.fnc

```diff
@@ -2064,14 +2065,16 @@ EXpRT	|I16	|do_uniprop_match|NN const char * const key|const U16 key_len
 EXpRT	|const char * const *|get_prop_values|const int table_index
 EXpR	|SV *	|get_prop_definition|const int table_index
 EXpRT	|const char *|get_deprecated_property_msg|const Size_t warning_offset
-#if defined(PERL_IN_REGCOMP_C)
-EiRT	|bool	|invlist_is_iterating|NN const SV* const invlist
-EiR	|SV*	|invlist_contents|NN SV* const invlist		    \
+#if defined(PERL_IN_REGCOMP_ANY)
+EpRT	|bool	|invlist_is_iterating|NN const SV* const invlist
+EpR	|SV*	|invlist_contents|NN SV* const invlist		    \
```

Comment:
You've changed some of the inline invlist functions to non-inline global functions, should they instead be moved to invlist_inline.h and remain inline?

@khwilliamson might have an opinion here.

Reply:
That is not quite correct. I change some file level inline functions to be available to the regex engine. Calling them "global" is misleading. They are still only visible in the regex engine.

If converting them from inline is a problem i will move them. Thanks for the review.

Reply:
Wont moving them to invlist_inline.h make them API implicitly? Not what we mark as 'A', but effectively there will be no protections for loading them IIUIR, as all the headers will be available to any module during build right? Anyway, ill make a stab at it and see how it works out.

Reply:
ohisee! @khwilliamson actually exercised discipline when he set up invlist_inline.h and it includes define guards for the files which use it. Nice. That is a bit different than the other inline files I have encounterd. Well done. ++

Reply:
K, that should all be sorted now. Sorry about that.

Reply:
> Calling them "global" is misleading

I struggled with the wording a bit.  They (were) global to the perl binary, but not API.  Perhaps I should have just said "no longer inline".

Reply:
Ah well, never mind. I was just feeling defensive. However you worded it, you were correct that it should be done a different way.